statsmodels/discrete/tests/test_discrete.py

```diff
@@ -1838,6 +1874,10 @@ def test_predict_prob(self):
         assert_allclose(chi2[:], (0.64628806058715882, 0.98578597726324468),
                         rtol=0.01)
 
+    def test_jac(self):
+        self.res1 = self.res
```

Comment:
Modifying `self` inplace here is a code smell.  Luckily there is a solution that goes hand-in-hand with the refactoring-out of `check_jac`: that function only needs `res1`, can get rid of the `self` argument and just pass `res1=self.res` and knock out two smells at once.